app/controllers/internal/mods_controller.rb

```diff
@@ -3,18 +3,26 @@ class Internal::ModsController < Internal::ApplicationController
 
   def index
     @mods = if params[:state] == "tag"
-              User.with_role(:tag_moderator, :any).page(params[:page]).per(50).includes(:notes)
+              User.with_role(:tag_moderator, :any).page(params[:page]).per(50)
             elsif params[:state] == "potential"
-              User.order("comments_count DESC").page(params[:page]).per(100).includes(:notes)
+              User.order("comments_count DESC").page(params[:page]).per(100)
             else
-              User.with_role(:trusted).page(params[:page]).per(50).includes(:notes)
+              User.with_role(:trusted).page(params[:page]).per(50)
             end
-    @mods = @mods.where("users.username ILIKE :search OR users.name ILIKE :search", search: "%#{params[:search]}%") if params[:search].present?
+
+    return unless params[:search]
```

Comment:
Do we want to add the `.present?` check here like we had before in case this is an empty string?

Reply:
I sometimes forget empty string is not falsey in Ruby :D